edx_proctoring/views.py

```diff
@@ -422,12 +422,18 @@ def put(self, request, attempt_id):
                     exam_id=attempt['proctored_exam']['id'],
                     user_id=request.user.id
                 )
-            elif action == 'submit':
+            elif action == 'timed_submit' or action == 'client_close':
                 exam_attempt_id = update_attempt_status(
                     attempt['proctored_exam']['id'],
```

Comment:
Why would `timed_submit` above be used with proctored_exam attempt?  Or does `timed_submit` not mean what I think?